indico/modules/events/management/util.py

```diff
@@ -182,3 +183,38 @@ def _process_attachments(folders):
 
     else:
         raise TypeError('Unexpected object of type {}: {}'.format(type(root).__name__, root))
+
+
+def find_unregistered_users(person_list):
+    """Get a set of users which don't have an Indico account
+
+        :param person_list: List of persons to check
+
+    """
+    return set(filter(lambda p: p.person.user is None, person_list))
```

Comment:
don't use filter with lambda, use a list comprehension instead. actually, here you want a set comprehension:

`{pl for pl in person_list if pl.person.user is None}`

I don't think we really need a function for it...